docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+
+services:
+  litecoind:
+    build: ./data-sources/litecoind
+    volumes:
+      - ./data-sources/litecoind/litecoin.conf:/home/litecoin/.litecoin/litecoin.conf
+    command: litecoind  --datadir=/home/litecoin/.litecoin --printtoconsole -rpcallowip=172.20.0.0/16
+  adapter:
+    image: node:8
+    environment:
+      - RPC_USERNAME=rpcuser
+      - RPC_PASSWORD=rpcpassword
+      - RPC_HOST=litecoind
+    working_dir: /home/node/app
+    volumes:
+      - /home/node/app/node_modules
```

Comment:
You can define the node_modules folder as its own volume so that you can do dev work locally without having to worry about a clash in npm installations.